src/app.js

```diff
@@ -1,21 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const ObjectToReturn = {
+    parts: [],
+    query: {},
+  };
+  const { url } = req;
+  const { host } = req.headers;
+  const normalizedUrl = new URL(url, `http://${host}`);
+
+  const { pathname } = normalizedUrl;
+  const parts = pathname.split('/').slice(1);
```

Comment:
you could involve the `url.parse` for this purpose
https://www.tutorialkart.com/nodejs/split-a-url-into-readable-parts-in-node-js/